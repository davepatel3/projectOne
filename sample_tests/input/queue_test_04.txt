int
10
4
2
ps
eb 51
ef -50
eb -42
df
eb 44
eb 71
ef 97
eb 72
pb
eb -98
ps
eb 43
ps
eb -11
ef 74
eb 67
ef -16
db
ps
gf
pf
pb
ef -37
gs
ef -95
ef -9
pf
ps
eb 17
ps
pf
ps
ef -4
pf
gc
pb
db
ps
ps
ef -76
eb -39
db
gc
eb -85
gs
ef 66
gf
eb -22
eb -2
pf
eb 66
ef -18
df
ps
df
ef -72
db
ps
df
ps
eb -19
db
gc
pf
db
pb
eb 12
ef -87
eb 22
gc
gc
gb
gc
eb -60
gc
eb -24
gs
df
ps
pf
ef -26
eb 93
db
gs
pb
eb -52
eb -22
eb 64
ps
eb 90
gc
pf
eb -46
gf
db
ef -47
db
pb
db
ps
eb 36
eb -80
df
pf
eb -31
eb 13
ps
eb -32
gs
gs
eb 97
pf
db
ps
ef 22
ef -12
db
eb -57
eb 88
cl
ef 1
pf
ef -18
ef -7
df
gs
gs
db
pb
gc
ef 8
ps
db
qt
