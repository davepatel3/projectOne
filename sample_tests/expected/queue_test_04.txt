[-][-][-][-][-][-][-][-][-][-]
-50
[+][+][+][+][+][-][-][-][-][+]
[+][+][+][+][+][+][-][-][-][+]
67
[+][+][+][+][+][+][+][+][+][-][-][-][-][-][-][-][-][-][-][-][-][-][-][-][-][-][-][-][-][-][-][-][-][-][-][-][-][-][-][+]
-16
9
[+][+][+][+][+][+][+][+][-][-][-][-][-][-][-][-][-][-][+][+]
[+][+][+][+][+][+][+][+][+][-][-][-][-][-][-][-][-][-][+][+]
[+][+][+][+][+][+][+][+][+][-][-][-][-][-][-][-][-][-][-][+]
20
43
[+][+][+][+][+][+][+][-][-][-][-][-][-][-][-][-][-][-][-][+]
[+][+][+][+][+][+][+][-][-][-][-][-][-][-][-][-][-][-][-][+]
-39
20
10
66
-18
[+][+][+][+][+][+][+][+][+][+][+][-][-][-][-][-][-][-][+][+]
-76
66
[+][+][+][+][+][+][+][+][+][+][-][-][-][-][-][-][-][-][+][+]
-72
[+][+][+][+][+][+][+][+][+][+][-][-][-][-][-][-][-][-][-][+]
-19
20
-2
20
20
22
20
20
13
-87
[+][+][+][+][+][+][+][+][+][+][+][+][-][-][-][-][-][-][-][-]
93
12
[+][+][+][+][+][+][+][+][+][+][+][+][+][+][-][-][-][-][-][-]
20
97
-46
90
-22
[+][+][+][+][+][+][+][+][+][+][+][+][-][-][-][-][-][-][-][-]
-47
[-][-][+][+][+][+][+][+][+][+][+][+][+][+][+][+][-][-][-][-]
15
15
97
[-][-][-][+][+][+][+][+][+][+][+][+][+][+][+][+][+][-][-][-]
-32
-7
1
1
-18
cyclic_double_queue is empty!
10
[-][-][-][-][-][-][-][-][+][-]
8
